testsuite/features/core_first_settings.feature

```diff
@@ -80,14 +80,15 @@ Feature: Very first settings
     And service "tomcat" is enabled on "server"
     And service "tomcat" is active on "server"
 
+  @http_proxy
```

Comment:
Maybe align to left, I think we put all @tags in column 1.

Reply:
Good point, changed :-)